File: DuDoong-Domain/src/main/java/band/gosrock/domain/domains/ticket_item/exception/TicketItemQuantityLackException.java
package band.gosrock.domain.domains.ticket_item.exception;

import band.gosrock.common.exception.DuDoongCodeException;

public class TicketItemQuantityLackException extends DuDoongCodeException {

    public static final DuDoongCodeException EXCEPTION = new TicketItemQuantityLackException();

    private TicketItemQuantityLackException() {super(TicketItemErrorCode.TICKET_ITEM_QUANTITY_LACK);}

}

File: DuDoong-Domain/src/main/kotlin/band/gosrock/domain/domains/ticket_item/exception/TicketItemQuantityLackException.kt
package band.gosrock.domain.domains.ticket_item.exception

import band.gosrock.common.exception.DuDoongCodeException

class TicketItemQuantityLackException private constructor() : DuDoongCodeException(TicketItemErrorCode.TICKET_ITEM_QUANTITY_LACK) {
    companion object {
        @JvmField
        val EXCEPTION: DuDoongCodeException = TicketItemQuantityLackException()
    }
}
